src/gui/guiButton.h

```diff
@@ -14,7 +14,7 @@
 
 using namespace irr;
 
-#if (IRRLICHT_VERSION_MAJOR > 1 || IRRLICHT_VERSION_MINOR != 8 || IRRLICHT_VERSION_REVISION < 5)
+#if (IRRLICHT_VERSION_MAJOR > 1 || IRRLICHT_VERSION_MINOR <= 8)

```

Comment:
2.0 is not <= 1.8
should be `#if (IRRLICHT_VERSION_MAJOR == 1 && IRRLICHT_VERSION_MINOR <= 8)`